Documentation/openssl.md

```diff
@@ -20,6 +20,14 @@ If you will be running a highly-available control-plane consisting of multiple m
 
 The IP address of the Kubernetes API Service. The `K8S_SERVICE_IP` will be the first IP in the `SERVICE_IP_RANGE` discussed in the [deployment guide][deployment-guide]. The first IP in the default range of 10.3.0.0/24 will be 10.3.0.1. If the SERVICE_IP_RANGE was changed from the default, this value must be updated as well.
 
+<hr/>
```

Comment:
I know this is just an addition here because it was moved, but we should try and use markdown instead (e.g. `---`)